postinstall.sh

```diff
@@ -6,9 +6,12 @@ set -x
 # If there is a libffmpegsumo, copy it over
 if test -f /opt/google/chrome/libffmpegsumo.so; then
 	cp /opt/google/chrome/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
+elif test -f /usr/lib/chromium/libffmpegsumo.so; then
+	cp /usr/lib/chromium/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
 else
 	echo "Couldn't locate libffmpegsumo.so during installation. Audio notifications will be disabled" 1>&2
-	echo "To repair this, please install Google Chrome and reinstall \`plaidchat\`" 1>&2
+	echo "To repair this, please install Google Chrome/Chromium and reinstall \`plaidchat\`" 1>&2
+	echo "Else you can copy the libffmpegsumo.so file in node_modules/nw/nwjs/" 1>&2
```

Comment:
i prefer an alert of this type to understand that i can insert the library manually.
In my case on debian sid chromium is not packaged with this library (i do know why) but i can find the library and place it manually.